packages/ts-config/main.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "main-tsconfig",
+  "compilerOptions": {
+      "target": "ESNext",
+      "moduleResolution": "node",
+      "module": "esnext",
+      "outDir": "dist",
+      "incremental": true,
+      "composite": true,
+      "declaration": true,
+      "resolveJsonModule": true,
+      "removeComments": true,
+      "emitDecoratorMetadata": true,
+      "experimentalDecorators": true,
+      "esModuleInterop": true,
+      "allowSyntheticDefaultImports": true,
+      "skipLibCheck": true,
+      "noImplicitAny": true,
+      "strictNullChecks": true,
+      "strictPropertyInitialization": false,
+      "strict": true,
+      "sourceMap": true,
+      "allowUnreachableCode": true,
```

nit: `allowUnreachableCode` parameter can lead to flaws in the code, as it allows the use of code that is unreachable in some cases. It is recommended to set it to `false` to improve code quality.

```suggestion
      "allowUnreachableCode": false,
```

---

I believe it's unnecessery ❤️